daml-lf/engine/src/test/scala/com/digitalasset/daml/lf/engine/ContractKeySpec.scala

```diff
@@ -378,30 +384,53 @@ class ContractKeySpec
         rollbackGlobalArchiveUpdates,
         rollbackGlobalArchivedLookup,
         rollbackGlobalArchivedCreate,
+        rollbackExerciseCreateFetchByKey,
+        rollbackExerciseCreateLookup,
+      )
+
+      val nonUckFailures = Set(
+        "RollbackExerciseCreateLookup",
+        "RollbackExerciseCreateFetchByKey",
       )
 
       val uckFailures = Set(
-        "CreateOverwritesLocal",
         "CreateOverwritesKnownGlobal",
+        "CreateOverwritesLocal",
+        "FetchDoesNotOverwriteGlobal",
+        "FetchDoesNotOverwriteLocal",
+        "GlobalArchiveOverwritesKnownGlobal1",
+        "GlobalArchiveOverwritesKnownGlobal2",
+        "GlobalArchiveOverwritesUnknownGlobal",
         "LocalArchiveOverwritesKnownGlobal",
         "RollbackCreateNonRollbackFetchByKey",
-        "RollbackFetchByKeyRollbackCreateNonRollbackFetchByKey",
+        "RollbackCreateNonRollbackGlobalArchive",
         "RollbackFetchByKeyNonRollbackCreate",
+        "RollbackFetchByKeyRollbackCreateNonRollbackFetchByKey",
+        "RollbackFetchNonRollbackCreate",
+        "RollbackGlobalArchiveNonRollbackCreate",
+        "RollbackGlobalArchiveNonRollbackCreate",
+        "RollbackGlobalArchiveUpdates",
```

Comment:
@andreaslochbihler-da @moritzkiefer-da could you double check that all those should actually fail.

Reply:
Indeed. All those should fail in mode Strict.

Reply:
failures look sensible to me.